about/research/index.html

```diff
@@ -45,14 +45,18 @@ <h3>{% t tech.academicpapers %}</h3>
                                   </div>
-								  <div class="col">
+				 	<div class="col">
                                       <a href="https://eprint.iacr.org/2022/288" alt="" class="paper-title">Spats: user-defined confidential assets for the Spark transaction protocol.</a>
                                       <p class="abstract">In privacy-preserving transaction protocols, confidential asset designs permit transfer of quantities of distinct asset types in a way that obscures their types and values. Spark is a protocol that provides flexible privacy properties relating to addressing, transaction sources and recipients, and value transfer; however, it does not natively support the use of multiple confidential asset types. Here we describe Spats, a new design for confidential assets compatible with Spark that focuses on efficient and modular implementation. It does so by extending coin value commitments to bind and mask an asset type, and asserting in zero knowledge that this type is maintained throughout transactions. We describe the cryptographic components and changes to the Spark protocol necessary for the design of Spats.</p>
                                   </div>
-								  <div class="col">
+					<div class="col">
                                       <a href="{{ site.baseurl }}/blog/assets/helsing-1049371.pdf" alt="" class="paper-title">Helsing: Private Masternode Staking</a>
                                       <p class="abstract">Helsing is a protocol extension to Spark that allows for private staking operations not requiring transparent addresses or outputs. Specifically, Helsing provides for Spark-compatible collateral staking and coinbase payouts.</p>
                                   </div>
+				      <dib class="col">
+					<a href="{{ site.baseurl }}/blog/assets/Lelantus_Sparks_with_Curve_Trees__Implementation_Notes.pdf" alt="" class="paper-title">Lelantus Spark with Curve Trees</a>

```

Comment:
```suggestion
					<a href="{{ site.baseurl }}/blog/assets/Lelantus_Sparks_with_Curve_Trees__Implementation_Notes.pdf" alt="implementation notes of Lelantus Spark with Curve Trees" class="paper-title">Lelantus Spark with Curve Trees</a>

```

In case an alt text is not provided, better not leave it blank using `alt=""`. It makes harder to identify missing alts with automatic tools and makes reading harder for people using accessibility software.